ls1intum/Artemis
# [Bugfix] Remove anchor titles with missing translations at exercise assessment dashboard information

### Checklist

### Motivation and Context


Solves issue #3889 

### Description

I don't think we need those titles/tooltips on the anchors, as the information table is already self-explanatory and we do not have such titles/tooltips in the information box on the course assessment dashboard.
I removed the titles and one the translation key, that was found but not used anywhere else.

### Steps for Testing


1. Log in to Artemis
2. Navigate to an exercise assessment dashboard and hover over the statistics, there should not be any anchor title (similar to tooltip) anymore

### Review Progress





- Code Review
- Manual Tests

### Screenshots


**This is how it should not be any more - those tooltips should not occur anymore**

https://user-images.githubusercontent.com/41366522/135289824-5830fcf0-b5c9-4616-a1f8-54e22c99abad.mov
